rosbag2_transport/src/rosbag2_transport/player.cpp

```diff
@@ -26,19 +26,16 @@
 #include "rcl/graph.h"
 #include "rcutils/time.h"
 
-#include "rosbag2_storage/serialized_bag_message.hpp"
-#include "rosbag2_storage/storage_interfaces/read_only_interface.hpp"
-#include "rosbag2_storage/storage_factory.hpp"
-#include "rosbag2/logging.hpp"
+#include "rosbag2_transport/logging.hpp"
 #include "rosbag2_node.hpp"
 #include "replayable_message.hpp"
-#include "typesupport_helpers.hpp"
+#include "rosbag2/typesupport_helpers.hpp"
 
-namespace rosbag2
+namespace rosbag2_transport
 {
 
-Player::Player(std::shared_ptr<rosbag2_storage::storage_interfaces::ReadOnlyInterface> storage)
-: storage_(storage), node_(std::make_shared<Rosbag2Node>("rosbag2_node"))
+Player::Player(std::shared_ptr<rosbag2::SequentialReader> reader)
+: reader_(std::move(reader)), node_(std::make_shared<Rosbag2Node>("rosbag2_node"))
```

Comment:
the node name should change here IMO. What about just `rosbag2_transport` ?

Reply:
If I understand correctly, you just mean the name as such, not so much the class name?
Sounds good, I'll change the name everywhere.